images/hook/entrypoint.sh

```diff
@@ -1,10 +1,25 @@
 #!/bin/sh
 set -e
 
-echo "Assuming changeset from the envrionment: $RIG_CHANGESET"
+echo "Assuming changeset from the environment: $RIG_CHANGESET"
 # note that rig does not take explicit changeset ID
 # taking it from the environment variables
-if [ $1 = "update" ]; then
+if [ $1 = "bootstrap" ]; then
+    echo "Checking: $RIG_CHANGESET"
+    if rig status $RIG_CHANGESET --retry-attempts=1 --retry-period=1s; then exit 0; fi
+
+    echo "Starting bootstrap, changeset: $RIG_CHANGESET"
+    # deleting in case it has been already attempted
+    rig cs delete --force -c cs/$RIG_CHANGESET
+    echo "Creating Log Forwarder ConfigMap"
+    rig upsert -f /var/lib/gravity/resources/logforwarder.yaml --debug
```

Comment:
If I'm reading this correctly, this will still reset the configuration map on install. What happens if the installation specifies logforwarder configuration via `--config` - in which case configuration will be created before the logging-app install hooks executes?
Do we need empty logforwarder configuration in the first place? IIRC, it will be crated on demand when adding/updating the configuration with `gravity resource create`.

Reply:
I'm pretty sure that if `rigging` find an existing resource will just replace it so if the configuration specified via `--config` is applied BEFORE the `logging-app` it's gonna be lost.

I don't know if we strictly NEED an empty container but I can definitely investigate.

Reply:
confirm that YES, the configmap is needed... 
```
Internal Error
configmaps "log-forwarders" not found, details: &StatusDetails{Name:log-forwarders,Group:,Kind:configmaps,Causes:[],RetryAfterSeconds:0,UID:,}
```

Reply:
creating an empty ConfigMap restores the functionality via WEB UI

Reply:
Does `--config` invoces the same creating of the ConfigMap? If it is the case then just make `if ! kubectl get configmap name >/dev/null 2>&1` around that rig upsert.

Reply:
yeah @a-palchikov was referring to that specific use-case.

I guess that testing if it exists and skipping is definitely the way to go.

Let me add that.